src/purchases.ts

```diff
@@ -732,17 +761,26 @@ export default class Purchases {
   }
 
   /**
-   * @deprecated, use enableAdServicesAttributionTokenCollection instead.
```

Comment:
I just noticed we currently have `syncObserverModeAmazonPurchase` above. We should deprecate that and move to the naming `syncAmazonPurchase` instead (That's what we did in the native side). I forgot this one on my PR. Would you be able to tackle that? I can take it if not.

Reply:
No problem! I've deprecated the existing one and replaced it with `syncAmazonPurchase()`